tests/metrics.test.ts

```diff
@@ -292,8 +293,8 @@ xdescribe('getMateAndAssists', () => {
   });
 });
 
-xdescribe('getMoveDistanceSingleGame', () => {
-  it('should return the correct max distance and piece for a game', async () => {
+describe('getMoveDistanceSingleGame', () => {
+  xit('should return the correct max distance and piece for a game', async () => {
```

Comment:
why canceling (or not) tests? For speed?

Reply:
easier to focus on one test at a time for me